packages/sdk/src/grpc/translation.ts

```diff
@@ -830,8 +841,8 @@ function trBakerEvent(bakerEvent: v2.BakerEvent, account: AccountAddress.Type):
                 account,
             };
         }
-        case undefined:
-            throw Error('Failed translating BakerEvent, encountered undefined');
+        default:
+            throw Error('Unrecognized event type. This should be impossible.');
```

Comment:
**Suggestion (non-blocking):** Won't this mean that we don't get compilation errors for non-exhaustive matching?

Reply:
True.. I didn't actually think that the ts compiler checked if all cases were checked in a switch. I'll try again 👍